### DISCUSSION TITLE
personal-development-aban-1402/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#26] [Announcements]
# personal-development-aban-1402/

طبق روال ماه‌های قبل، گزارش توسعه فردی این ماه رو منتشر می‌کنم.
آموزشی سربازی از ابتدای این ماه تا همین دیروز درگیر دوره آموزشی بودم. تو پادگان ۰۱ ارتش دوره آموزشی رو گذروندم و انصافاً دوره‌ای سرتاسر بیهوده و عبث بود. یعنی جوری برامون برنامه چیده بودند که از صبح تا شب مشغول باشیم ولی هیچ کاری نکنیم.
تنها نکته مثبتش پیداکردن دوست بود. دوست‌هایی از حوزه‌های مختلف؛ پزشک، جراح قلب، مدیرعامل، برنامه‌نویس، معامله‌گر بورس.

https://aprd.ir/personal-development-aban-1402/

**Comment by HashemAlavi** [comment]
سلام امیر جان. امیدوارم حالت خوب باشه:) و بتونی این خدمت (عبث) سربازی رو به سلامتی تمومش کنی.

واقعاً بعضی از پدیده‌هایی که تو ایران وجود داره مایه سردرد و تعجب منه. یکیش هم این سربازیه. هر چند تو برنامه هفتم توسعه (یا شایدم عدم توسعه) قرار شد این مدت به 12 ماه کاهش پیدا کنه؛ اما فکر می‌کنم که حتی 1 ماه هم زیاده براش. چه برسه به 1 سال. موندم که خودم چجوری می‌تونم تحملش کنم بعداً.

در مورد پادکست ایستگاه هوش مصنوعی: راستش تو این مدت یکم درگیر حرف و حدیث‌هایی بودم که در مورد اثرات هوش مصنوعی روی مشاغل و دنیای آینده زده میشه. منم که بیسواد و این حرف‌ها می‌رفت رو مخم. و نگران میشدم از اینکه بعداً قراره چه بلایی سرم بیاد. فکر کنم بهترین راه اینه که یکم اطلاعات بیشتری راجع به این حوزه داشته باشم و مثل اینکه الآن منبع خوبی براش پیدا کردم:)

**Reply by pourmand1376** [reply]
سلام هاشم جان. خیلی ممنون از لطفت. 

من سربازی رو با [طرح صیاد شیرازی](https://bmn.ir/%D8%B7%D8%B1%D8%AD-%D8%B4%D9%87%DB%8C%D8%AF-%D8%B5%DB%8C%D8%A7%D8%AF-%D8%B4%DB%8C%D8%B1%D8%A7%D8%B2%DB%8C) گذروندم و این‌طور که می‌گفتند ما بچه‌های آبان، «سرباز نخبه» بودیم. البته من با لفظ نخبه مشکل دارم ولی خب خوبیش این بود که آموزشی ما نسبت به آموزشی عادی خیلی راحت‌تر بود و یک ماهه هم تموم شد. البته به نظر من که یک ماه هم زیاد بود و همه چیزهایی که گفتند رو میشه در ۳ روز بصورت فشرده آموزش داد. 

از الان هم دیگه کاری با پادگان و یگان و این حرف‌ها ندارم. صرفاً میرم با استاد دانشگام روی یک پروژه‌ هوش‌مصنوعی که تعریف کردیم، کار می‌کنم و تقریباً یه سال دیگه سربازی‌ام تموم میشه. 

به نسبت بقیه مسیرهای سربازی این روش خیلی بهتره. نکته مثبتش اینه که معمولاً فقط باید پروژه تعریف‌شده انجام بشه و مهم نیست تو خونه انجام بدی یا اصلاً سرکار بری و انجام بدی. سالانه هم هزاران نفر از همین روش کارت پایان خدمت می‌گیرند و اصلاً چیز عجیبی نیست. خواستی بیشتر بدونی، [این کانال](https://t.me/tarhe_shahid_sayyad_bonyad) هست. 

تو پادکست هم البته هنوز هیچی نگفتم. ان‌شاءالله از این ماه بیشتر روش کار می‌کنم. فکر کنم ماهی دو تا قسمت مناسب باشه.
